modules/sdk/src/main/scala/org/tessellation/sdk/domain/trust.scala

```diff
@@ -0,0 +1,30 @@
+package org.tessellation.sdk.domain
+
+import cats.syntax.bifunctor._
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.{InternalTrustUpdate, InternalTrustUpdateRefined, InternalTrustUpdateRefinement}
+import org.tessellation.security.hex.Hex
+
+import derevo.cats.eqv
+import derevo.derive
+import eu.timepit.refined.cats._
+import eu.timepit.refined.refineV
+import fs2.data.csv.generic.semiauto.deriveRowDecoder
+import fs2.data.csv.{CellDecoder, DecoderError}
+
+object trust {
+  @derive(eqv)
+  case class TrustRating(peerId: PeerId, rating: InternalTrustUpdateRefined) {
```

Comment:
Do these two classes have semantically different uses despite looking identical?
```
case class InternalTrustUpdate(id: PeerId, trust: InternalTrustUpdateRefined)
```

Reply:
Good point

Reply:
@marcinwadon @sdavidp , yes. @ryle-goehausen wants to keep them separate because one is internal facing and the other is external facing.

Reply:
What is the use case for both of them then? For now you don't use one, it you just map one into another. I'm in favor of abstracting the external one only if there is a need for that.

Reply:
The two classes have been deduplicated for now. When they diverge, they will be changed then.